app/assets/stylesheets/styleguide/_font-stacks.scss

```diff
@@ -32,6 +32,6 @@ $NTA-Light:
        local("Helvetica");
 }
 
-$Helvetica-Regular:
+$helvetica-regular:
```

Comment:
Looks like this variable was removed from _organisations.scss - is it still needed?

Reply:
I've removed this file altogether. Could not find a reference to it anywhere (I don't think it's actually been used in a long while?). The most recent changes to this file were from 8 years ago 😬

Reply:
Good shout. Although it's worth pointing out that even if something's not been updated for a while it doesn't mean we're not still depending on it! Hence my earlier enthusiasm for removing much of this stuff...